game/src/edit/roads.rs

```diff
@@ -422,19 +422,25 @@ fn make_main_panel(
     ]
     .into_iter()
     .map(|(lt, key)| {
-        let disabled = if let Some(current) = current_lt {
-            lt == current
-        } else {
-            // Only 2 parking lanes or walkable lanes total make sense
-            (lt == LaneType::Parking
+        // If the selected lane is already this type, we can't change it.
+        let disabled = Some(lt) == current_lt
```

Comment:
A disabled button doesn't display a tooltip right now, but I'm thinking of changing that, so we can explain these rules when you hover over the greyed-out button.